Packages/Env/Sources/Env/Router.swift

```diff
@@ -62,9 +62,9 @@ public class RouterPath: ObservableObject {
   }
 
   public func handleStatus(status: AnyStatus, url: URL) -> OpenURLAction.Result {
-    if url.pathComponents.contains(where: { $0 == "tags" }),
-       let tag = url.pathComponents.last
+    if url.pathComponents.count == 3 && url.pathComponents[1] == "tags"
     {
+      let tag = url.pathComponents.last!
```

Comment:
I'm not a maintainer here, but I find force unwraps to be _bad_. The previous code used `guard` to ensure the tag could be extracted.

Looking at the Issue this is set to fix, I'm uncertain why `count == 3` and `pathComponents[1]` are important preconditions. 

Can you please explain the logic to me?

Reply:
The existing code matches for tags anywhere in the path so `/foo/tags/bar/` and `/tags/foo/bar/` and just `/tags/` are all matches.  Then it plucks the last part off the path and makes that the navigation target for the tag view.

However, only paths in the form `/tags/name_of_tag` are actually valid, so that's the test here.

3 pathComponents, namely `[ "/", "tags", "name_of_tag"]`. 
`pathComponent[1]` has to be `"tags"` or it's not a match, so not `/foo/tags` for example 
The force unwrap is guaranteed to work because we know there are 3 components so `last` will definitely have a value.

Reply:
Having said that, I have no strong feelings about force unwraps, so I've changed it back to the style it was before.